backend/BjjAppApi/obj/project.nuget.cache

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
is this cache file needed? maybe something we can add to the git ignore?

Reply:
Good catch. We don't need that file. The .gitignore has the rule for it but file was already tracked before the .gitignore was created.